Console.rb

```diff
@@ -0,0 +1,61 @@
+# Utility for writing to terminal, mostly colors and cursor movements.
+
+module Console
+	def self.color(color, text)
+		return text if color.nil?
+		self.public_send(color, text)
+	end
```

Comment:
Good point. It looks fine in my version in sublime, but odd here, will have to see what's going on.